domains/graph/components/ProfileAssetsGraph.vue

```diff
@@ -132,16 +132,18 @@ const handleChangeType = async (customEvent: CustomEvent) => {
 }
 
 const handleChangeCollection = async (customEvent: CustomEvent) => {
-  const value = customEvent.detail?.value
+  const collectionAddress = customEvent.detail?.value?.id as string
 
-  if (filters.collections?.includes(value)) {
+  if (filters.collections?.includes(collectionAddress)) {
     setFilters({
       collections: filters.collections?.filter(
-        collection => collection !== value
+        collection => collection !== collectionAddress
```

Comment:
Wondering if we could use `Set<string>` for `filters.collections` and maybe other filters as well to avoid using `.filter(...)`.
Do we care about the order in those filters?

Reply:
Yes we care about order.